packages/core/src/modules/proofs/protocol/v1/messages/V1RequestPresentationMessage.ts

```diff
@@ -65,17 +67,25 @@ export class V1RequestPresentationMessage extends AgentMessage {
   @IsInstance(Attachment, { each: true })
   public requestPresentationAttachments!: Attachment[]
 
-  public get indyProofRequest(): ProofRequest | null {
+  public get indyProofRequest(): IndyProofRequest | null {
     const attachment = this.requestPresentationAttachments.find(
       (attachment) => attachment.id === INDY_PROOF_REQUEST_ATTACHMENT_ID
     )
     // Extract proof request from attachment
-    const proofRequestJson = attachment?.getDataAsJson<ProofRequest>() ?? null
+    const proofRequestJson = attachment?.getDataAsJson<IndyProofRequest>() ?? null
```

Comment:
We can now replace this with `const proofRequestJson = this.indyProofRequestJson`